draft-ietf-httpbis-digest-headers.md

```diff
@@ -86,13 +85,13 @@ informative:
 
 --- abstract
 
-This document defines the Digest and Want-Digest header fields for HTTP, thus
-allowing client and server to negotiate an integrity checksum of the exchanged
-resource representation data.
+This document defines the HTTP Digest and Want-Digest fields, thus allowing
+client and server to negotiate an integrity checksum of the exchanged resource
+representation data.
 
 This document obsoletes RFC 3230. It replaces the term "instance" with
-"representation", which makes it consistent  with the HTTP Semantic and Context
-defined in RFC 7231.
+"representation", which makes it consistent with the HTTP Semantic and Context
+defined in draft-ietf-httpbis-semantics.
```

Comment:
```suggestion
"representation", which makes it consistent with the HTTP Semantic and Context.
```
Referencing the draft or RFC does not add anything here.

Reply:
I disagree. The main purpose of this document is to update Digest to align with semantics, and IMO we should be abundantly clear exactly which version of semantic we are aligning to.